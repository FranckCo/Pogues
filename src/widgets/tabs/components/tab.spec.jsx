import React from 'react';
import renderer from 'react-test-renderer';

import Tab from './tab';

describe('Tab', () => {
  test('Should exists with the corresponding template', () => {
    const props = {
      label: 'Fake label',
      path: 'fake path',
    };
    const tab = renderer
      .create(
        <Tab {...props}>
          <div>Fake Children</div>
        </Tab>,
      )
      .toJSON();

    expect(tab).toMatchSnapshot();
  });
});
