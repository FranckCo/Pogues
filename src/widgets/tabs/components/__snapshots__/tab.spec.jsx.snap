// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tab > Should exists with the corresponding template 1`] = `
<div
  className="nav-content"
>
  <div>
    Fake Children
  </div>
</div>
`;
